wee_slack.py

```diff
@@ -1444,7 +1444,8 @@ def process_im_open(message_json):
 def process_im_marked(message_json):
     channel = channels.find(message_json["channel"])
```

Comment:
When doing this, I noticed that this method uses `channels` directly, while the two methods above does `server = servers.find...` and then uses `server.channels`. Should this method do the same?

(This comment is not really related to this PR though.)